module/x/gravity/keeper/attestation_handler.go

```diff
@@ -35,7 +35,9 @@ func (a AttestationHandler) Handle(ctx sdk.Context, att types.Attestation, claim
 	switch claim := claim.(type) {
 
 	case *types.MsgSendToCosmosClaim:
-		return a.handleSendToCosmos(ctx, *claim)
+		// Don't handle sendToCosmos claims. We still store them (so we can keep track of the event nonce and don't
+		// block other claims) in state but we don't execute them.
+		return nil
```

Comment:
We shouldn't block it, we need to ingest it and ignore it. Because if we don't, we would be blocking the event nonce. For example, a SendToCosmos has event nonce 2 and it gets blocked, then a ValsetUpdate comes in with nonce 3 and it won't get accepted because we are missing nonce 2